src/handlers/ motions/helpers.ts

```diff
@@ -0,0 +1,135 @@
+import Decimal from 'decimal.js';
+import { BigNumber } from 'ethers';
+import { mutate } from '~amplifyClient';
+import { MotionQuery, MotionSide, UserStake } from '~types';
+import { convertStakeToPercentage } from '~utils/motions';
+
+const getUpdatedUserStakesAmount = (
```

Comment:
I'm really lacking some comments here. Maybe it's because it's all new stuff to me, but also the names of the functions are very similar so some documentation to understand what's going on (high-level) would be great